docs/arch/scheduler.md

```diff
@@ -0,0 +1,74 @@
+# Scheduler
+
+## Contributors
+
+- DE MARTINO Giada Flora
+- DURAT Mathias
+- PLANCHE Benoit
+- TCHILINGUIRIAN Théo
+
+
+## Lexicon
+
+- Step: a step is a CI job unit.
+- Agent: an agent is a computing node registered with the scheduler. They constitute a pool of available computing resources.
```

Comment:
One agent is on agent. The pool of agent is something else and it could be developed else where

Reply:
outdated. Is the current documentation up to your standards ?